VDA-XP-Reward/.env.example

```diff
@@ -0,0 +1,17 @@
+# Used to verify contract after deployment
+ETHERSCAN_API_KEY="<Input your etherscan.com api key>"
+POLYGONSCAN_API_KEY="<Input your polygonscan.com api key"
```

Comment:
Add a comment describing why this is needed

Reply:
Pls see the line#1.
This is used to verify the contract after deployment.